lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/openturns/ProjectionStrategyImplementation.hxx

```diff
@@ -151,6 +165,12 @@ protected:
 
   // Proxy to speed up evaluations of the basis over the input sample
   DesignProxy proxy_;
+
+  // isLeastSquares_ is true if the projection is a regression
+  Bool isLeastSquares_;
+
+  // isModelSelection flag
+  Bool isModelSelection_;
```

Comment:
they could be initialized here instead of several times in the .cxx

Reply:
```suggestion
  Bool isModelSelection_ = false;
```